src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -593,6 +591,71 @@ private void finishImportList() {
         }
     }
 
+    /**
+     * Get first import group.
+     *
+     * @return
+     *        first import group of file.
+     */
+    private String getFirstGroup() {
+        final ImportDetails firstImport = importToGroupList.get(0);
+        return getImportGroup(firstImport.isStaticImport(),
+                firstImport.getImportFullPath());
+    }
+
+    /**
+     * Examine alphabetical order of imports.
+     *
+     * @param previousImport
+     *        previous import of current group.
+     * @param fullImportIdent
+     *        full import ident.
+     * @return
+     *        true, if previous and current import are not in alphabetical order.
+     */
+    private boolean isAlphabeticalOrderBroken(String previousImport,
+                                              String fullImportIdent) {
+        return sortImportsInGroupAlphabetically
+                && previousImport != null
+                && compareImports(fullImportIdent, previousImport) < 0;
+    }
+
+    /**
+     * Examine empty lines between groups.
+     *
+     * @param previousImportObject
+     *        previous import in current group.
+     * @param importObject
+     *        import group.
```

Comment:
> import group
> if current import NOT separated from previous import 

Javadoc definitions do not match up. This one should change to current import and variable should be renamed to as such.

Reply:
done